manifests/charts/istiod-remote/values.yaml

```diff
@@ -0,0 +1,444 @@
+revision: ""
+# Revision tags are aliases to Istio control plane revisions
+revisionTags: []
+# For Helm compatibility.
+ownerName: ""
+# meshConfig defines runtime configuration of components, including Istiod and istio-agent behavior
+# See https://istio.io/docs/reference/config/istio.mesh.v1alpha1/ for all available options
+meshConfig:
+  enablePrometheusMerge: true
+  # Config for the default ProxyConfig.
+  # Initially using directly the proxy metadata - can also be activated using annotations
+  # on the pod. This is an unsupported low-level API, pending review and decisions on
+  # enabling the feature. Enabling the DNS listener is safe - and allows further testing
+  # and gradual adoption by setting capture only on specific workloads. It also allows
+  # VMs to use other DNS options, like dnsmasq or unbound.
+
+  # The namespace to treat as the administrative root namespace for Istio configuration.
+  # When processing a leaf namespace Istio will search for declarations in that namespace first
+  # and if none are found it will search in the root namespace. Any matching declaration found in the root namespace
+  # is processed as if it were declared in the leaf namespace.
+  rootNamespace:
+  # The trust domain corresponds to the trust root of a system
+  # Refer to https://github.com/spiffe/spiffe/blob/master/standards/SPIFFE-ID.md#21-trust-domain
+  trustDomain: "cluster.local"
+  # TODO: the intent is to eventually have this enabled by default when security is used.
+  # It is not clear if user should normally need to configure - the metadata is typically
+  # used as an escape and to control testing and rollout, but it is not intended as a long-term
+  # stable API.
+# What we may configure in mesh config is the ".global" - and use of other suffixes.
+# No hurry to do this in 1.6, we're trying to prove the code.
+
+global:
+  # Used to locate istiod.
+  istioNamespace: istio-system
+  # enable pod disruption budget for the control plane, which is used to
+  # ensure Istio control plane components are gradually upgraded or recovered.
+  defaultPodDisruptionBudget:
+    enabled: true
+    # The values aren't mutable due to a current PodDisruptionBudget limitation
+    # minAvailable: 1
+  # A minimal set of requested resources to applied to all deployments so that
+  # Horizontal Pod Autoscaler will be able to function (if set).
+  # Each component can overwrite these default values by adding its own resources
+  # block in the relevant section below and setting the desired resources values.
+  defaultResources:
+    requests:
+      cpu: 10m
+      #   memory: 128Mi
+      # limits:
+      #   cpu: 100m
+      #   memory: 128Mi
+  # Default hub for Istio images.
+  # Releases are published to docker hub under 'istio' project.
+  # Dev builds from prow are on gcr.io
+  hub: gcr.io/istio-testing
+  # Default tag for Istio images.
+  tag: latest
```

Comment:
This was included in the 1.11 release. Isn't it supposed to be the following just like the previous version?
```
  # Default hub for Istio images.
  # Releases are published to docker hub under 'istio' project.
  # Dev builds from prow are on gcr.io
  hub: docker.io/istio
  # Default tag for Istio images.
  tag: 1.11.0
```

Reply:
No, our git source always has `latest`. When we build a release we plug in the appriopriate hub/tag

Reply:
Ah ok thanks for the clarification. But it seems like in the latest release(1.11.0) the test hub/tag were somehow used instead of the appropriate values.  
I tried  istio-1.11.0-linux-amd64.tar.gz at https://github.com/istio/istio/releases.

Reply:
Thanks @marsblkm. This is a bug in the release-builder. We're missing the istiod-remote chart in the list of charts to stanitize: https://github.com/istio/release-builder/blob/530eacb6a52d45f80f41be643530d94792088287/pkg/build/helm.go#L87. Opened issue: https://github.com/istio/istio/issues/34804.